src/App/FreeCADInit.py

```diff
@@ -696,16 +701,35 @@ def report(self,msg,func,*args,**kargs):
 App.Units.VoltAmpere    = App.Units.Quantity('VA')
 
 App.Units.Volt          = App.Units.Quantity('V')
+App.Units.MilliVolt     = App.Units.Quantity('mV')
+App.Units.KiloVolt      = App.Units.Quantity('kV')
+
+App.Units.Coulomb       = App.Units.Quantity('C')
+
+App.Units.Tesla         = App.Units.Quantity('T')
+
+App.Units.PicoFarad     = App.Units.Quantity('pF')
+App.Units.NanoFarad     = App.Units.Quantity('nF')
+App.Units.MicroFarad    = App.Units.Quantity('uF')
+App.Units.MilliFarad    = App.Units.Quantity('mF')
+App.Units.Farad         = App.Units.Quantity('F')
+
+App.Units.NanoHenry     = App.Units.Quantity('nH')
+App.Units.MicroHenry    = App.Units.Quantity('uH')
+App.Units.MilliHenry    = App.Units.Quantity('mH')
+App.Units.Henry         = App.Units.Quantity('H')
 
 App.Units.Joule         = App.Units.Quantity('J')
+App.Units.KiloJoule     = App.Units.Quantity('kJ')

```

Comment:
Should we add millijoule as well? Maybe it's useful for some small scale analysis of integrated circuits or something like that. I'm not sure of the scale of the units but I think electronics are sometimes designed considering power consumption in milliwats, so millijoule would seem appropriate.

Reply:
> Should we add millijoule as well?

Yes. I have more additions in the pipeline but the patch is already very big. So let's add further units in a next step.

Reply:
> Should we add millijoule as well?

OK, I added this now and also kW